posthog/api/organization_member.py

```diff
@@ -71,7 +71,9 @@ class OrganizationMemberViewSet(
 ):
     serializer_class = OrganizationMemberSerializer
     permission_classes = [IsAuthenticated, OrganizationMemberPermissions, OrganizationMemberObjectPermissions]
-    queryset = OrganizationMembership.objects.exclude(user__email__endswith=INTERNAL_BOT_EMAIL_SUFFIX)
+    queryset = OrganizationMembership.objects.exclude(
```

Comment:
Note that this will be excluding memberships where `user__email__endswith=INTERNAL_BOT_EMAIL_SUFFIX` _and_ `user__is_active=False`

I suspect we want an or here?

Reply:
Oops, yeah, wanted to simplify this a moment ago but indeed made it incorrect. Fixed.